public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
-    ```
-    npm init -y
-    npm i angular2@2.0.0-alpha.44 systemjs@0.19.2 --save --save-exact
-    npm i typescript live-server --save-dev
-    ```
-
-    These commands both *install* the packages and *create* an npm `package.json` that will
-    help us develop and maintain our application in the future.
-    The essence of our `package.json` should look like this:
+:marked
+  ## Add the `index.html`
+  
+  Angular displays our application in a specific location on our `index.html`.
+  It's time to create that file.
+
+  We won't put our `index.html` in the `app/` folder. 
+  We'll locate it **up one level, in the project root folder**.
+  
+code-example(format="").
+  cd ..
+:marked
+  Now create the`index.html` file and paste the following lines:
++makeExample('quickstart/ts/index.html', null, 'index.html')
+:marked
+  There are three noteworthy sections of HTML:
 
-  +makeJson('quickstart/ts/package.json', { paths: 'name, version, dependencies, devDependencies'})
+  1. We load the JavaScript libraries we need.<br/>
 
-  :marked
-    There is also a `scripts` section. **Find and replace** it with the following:
+  2. We configure something called `System` and ask it to import the
+  boot file we just wrote.
 
-  +makeJson('quickstart/ts/package.json', { paths: 'scripts'})
-
-  :marked
-    We've just extended our project world with script commands
-    that we'll be running very soon.
+  3. We add the `<my-app>` tag in the `<body>`. **This is where our app lives!**
+  
+  Something has to find and load our application modules. We're using **SystemJS** to do that.
+  There are other choices and we're not saying SystemJS is the best. We like it and it works.
+  
+  The specifics of SystemJS configuration are out of bounds. 
+  We'll briefly describe this particular configuration in the [appendix below](#systemjs).
+  
+  When Angular calls the `bootstrap` function in `boot.ts`, it reads the `AppComponent`
+  metadata, finds the `my-app` selector, locates an element tag named `my-app`,
+  and loads our application between those tags.
 
 .l-main-section
-  :marked
-    ## Update `index.html`
-
-    **Replace** the library scripts section with references to
-    scripts in the packages we just installed.
-
-  +makeExample('quickstart/ts/src/index.html', 'libraries')
-
-  :marked
-    **Update** the `System` configuration script as follows.
-
-  +makeExample('quickstart/ts/src/index.html', 'systemjs')
-
-  .l-sub-section
-    :marked
-      We won't be transpiling TypeScript in the browser anymore.
-      We'll do that on our machine and ship the generated JavaScript
-      files to the server.
-
-      We have to re-configure `system.js` to expect JavaScript files
-      with a `.js` extension by default.
-      Someday we might add a `Foo` class to our application in a `foo.ts`
-      file and import it like this
+:marked
+  ## Compile and run!
+  
+  Open a terminal window and enter this command:
+code-example(format="").
+  npm run both
```

Comment:
This does not work for me after running `gulp add-example-symlinks` at `angular.io/`. 

I think you're missing these packages from the root `angular.io/package.json` (as they are present in `package.1.json` in the quickstart example)

```
"concurrently": "^1.0.0",
"lite-server": "^1.3.1",
```

This is the error I got:

```
filipe.silva@DESKTOP-7ND6T3R MINGW64 /E/dev/ideablade/angular.io/public/docs/_examples/quickstart/ts (quickstart-51)
$ npm run both

> angular2-quickstart@1.0.0 both E:\dev\ideablade\angular.io\public\docs\_examples\quickstart\ts
> concurrent "npm run tsc:w" "npm run lite"

'concurrent' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "both"
npm ERR! node v4.2.1
npm ERR! npm  v2.14.7
npm ERR! code ELIFECYCLE
npm ERR! angular2-quickstart@1.0.0 both: `concurrent "npm run tsc:w" "npm run lite" `
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the angular2-quickstart@1.0.0 both script 'concurrent "npm run tsc:w" "npm run lite" '.
npm ERR! This is most likely a problem with the angular2-quickstart package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     concurrent "npm run tsc:w" "npm run lite"
npm ERR! You can get their info via:
npm ERR!     npm owner ls angular2-quickstart
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     E:\dev\ideablade\angular.io\public\docs\_examples\quickstart\ts\npm-debug.log
```